src/http-util/src/lib.rs

```diff
@@ -128,3 +129,14 @@ pub async fn handle_modify_filter_target(
         Err(e) => (StatusCode::BAD_REQUEST, e.to_string()),
     }
 }
+
+/// Allows dynamic control of the stderr log filter
+#[allow(clippy::unused_async)]
+pub async fn generate_tracing_info() -> impl IntoResponse {
+    (
+        StatusCode::OK,
+        Json(json!({
+            "filter_fast_path": tracing::level_filters::LevelFilter::current().to_string()
```

Comment:
Should this be called `current_level_filter` to match the Rust API?

```suggestion
            "current_level_filter": tracing::level_filters::LevelFilter::current().to_string()
```